recipe/build.sh

```diff
@@ -0,0 +1,5 @@
+$PYTHON -m pip install . --no-deps -vv
+
+cd js
+$PYTHON -m pip install . --no-deps -vv
+
```

Comment:
This conda package is installing two different python libs here. I am not sure this considered good practice on conda-forge (but I might be wrong).

Would that be possible to merge both `setup.py` and `js/setup.py` files into a single one?

ping @conda-forge/help-python to get more information here.

Reply:
> Would that be possible to merge both setup.py and js/setup.py files into a single one?

We are thinking about that too so `pip` users won't need to install `nglview-js-widgets`
https://github.com/nglviewer/nglview/issues/964

Reply:
I'd suggest this route over two setup.py files.

Reply:
@jan-janssen Will `conda uninstall nglview` remove the nglview-js-widgets `js` files too?

Reply:
@hainm yes

Reply:
@hadim I created a different pull request which creates two separate packages: 
https://github.com/conda-forge/nglview-feedstock/pull/51

Again just a suggestion

Reply:
It's fine to merge here (and close #51). Once both setup.py are merged and https://github.com/nglviewer/nglview/issues/964 closed, we can bump here and go back to a normal single `$PYTHON -m pip install . --no-deps -vv`.

Before merging can you add an import test for the js package?

Reply:
Done